ciri: cond_handle_timedwait warning

**Describe the bug**
cIRI compilation raises a warning.

**To Reproduce**
Steps to reproduce the behavior:
1. Compile cIRI

**Expected behavior**
```
In file included from ./gossip/components/broadcaster.h:15:0,
                 from ./gossip/node.h:12,
                 from gossip/services/tcp_receiver.cc:11:
./utils/handles/cond.h: In function 'int cond_handle_timedwait(cond_handle_t*, lock_handle_t*, uint64_t)':
./utils/handles/cond.h:56:18: warning: comparison between signed and unsigned integer expressions [-Wsign-compare]
   if (ts.tv_nsec >= 1000000000ULL) {
       ~~~~~~~~~~~^~~~~~~~~~~~~~~~
```

**Desktop (please complete the following information):**
 - Linux

--- a/utils/handles/cond.h
+++ b/utils/handles/cond.h
@@ -53,7 +53,7 @@ static inline int cond_handle_timedwait(cond_handle_t* const cond, lock_handle_t
   gettimeofday(&tv, NULL);
   ts.tv_sec = tv.tv_sec + timeout_ms / 1000ULL;
   ts.tv_nsec = tv.tv_usec * 1000ULL + (timeout_ms % 1000ULL) * 1000000ULL;
-  if (ts.tv_nsec >= 1000000000ULL) {
+  if (ts.tv_nsec >= 1000000000LL) {
     ts.tv_sec++;
     ts.tv_nsec -= 1000000000ULL;
   }

